<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusWindow</class>
 <widget class="QWidget" name="BusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 252, 255);</string>
  </property>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>42</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bus2</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="boderView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>264</width>
     <height>84</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/otherImage/images/chatUI.png);
background-color: rgb(149, 255, 232);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>42</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bus1</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textDisplay">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>60</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <zorder>boderView</zorder>
  <zorder>textDisplay</zorder>
  <zorder>rightButton</zorder>
  <zorder>leftButton</zorder>
  <zorder>endButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
